ta4j-core/src/main/java/org/ta4j/core/BaseStrategy.java

```diff
@@ -121,6 +129,11 @@ public Rule getExitRule() {
     public int getUnstablePeriod() {
     	return unstablePeriod;
     }
+	
+    @Override
+    public void setHint(String hint) {
+    	this.hint = hint;
+    }
```

Comment:
Field is not necessary (at least not in the scope of this PR "Add: Z-Score-Indicator")